### ISSUE TITLE
Transaction Sent to CCB already exist!

### ISSUE THREAD

**Issue by pmt-it** [pmt-it/Admin#32]
If you look at [INTEGRATION_IOMessages] table for transaction 10100000182 and ID = 50953. You see we have send them the request but there is no response from CCB (maybe it got timeout) then when we resend the txn to them, it resulted in duplication of TransactionID.

I think we need to increase the timeout value. What do you think?

select \* from [dbo].[INTEGRATION_IOMessages] where payoutreferenceid = '10100000182' order by datetime

![image](https://cloud.githubusercontent.com/assets/6502098/4403409/bc111eb0-44a2-11e4-86d2-8d23fe26ca79.png)

**Comment by jctopo** [comment]
This is not a bug.

For now we are assuming this is due to a timeout, but could be anything else. The problem is that we dont have a usefull response code that tell us this transaction is duplicate, if we would have it, then we can procede to set the transaction as Ready For Payment.

This transaction is already paid, problem is CEG doesnt notice yet.

**Comment by pmt-it** [comment]
Yes. This could be anything else but we cannot just leave it and do nothing. What do you propose we need to do if we don't get a response for the request we send out.
